core/drivers/crypto/caam/acipher/caam_rsa.c

```diff
@@ -222,7 +222,7 @@ static enum caam_status do_keypair_conv_f3(struct caam_rsa_keypair *outkey,
 			outkey->dp.length + outkey->dq.length +
 				outkey->qp.length);
 
-	outkey->format = 2;
+	outkey->format = 3;
```

Comment:
It would be nice a descriptive define instead.

Reply:
+1 and in the commit description too (minor note: avoid mentioning #3 because Github create links automatically; say "number 3" instead)

Reply:
Good idea, please see last commit added

Reply:
@jforissier Oh! I will fix it